include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -955,7 +955,7 @@ is supported.
 
 // ----------------------------------- version = 1 ----------------------------------- //
 
-Row |                  warp 0                                    warp 2
+Row |                  warp 0                                    warp 1
```

Comment:
@joviliast 
FYI, I've looked into `getWarpOrder` implementation it looks like this now.